package.json

```diff
@@ -40,6 +40,7 @@
     "@google-cloud/common": "^0.11.0",
     "continuation-local-storage": "^3.1.4",
     "gcp-metadata": "^0.1.0",
+    "is": "^3.2.0",
```

Comment:
Also note that `is.object() !== _.isObject()`, they will result in different results. e.g. `is.object([]) === false` while `_.isObject([]) === true`